scripts/cluster-upgrade.sh

```diff
@@ -1,19 +1,75 @@
-#!/bin/bash
-# This script contains non-portable components and is intended to run on a Hive cluster.
+#!/bin/bash -e
 
-OCP_VERSION_FROM=$1
-OCP_VERSION_TO=$2
+STATUS=false
+SCRIPT_NAME=$(basename "$0")
+
+usage() {
+    cat <<EOF
```

Comment:
Reason for cat > echo?

Reply:
Its cleaner, otherwise I'd have to echo every line or string together `\n`'s. Do you think?

Reply:
Oh yeah, you're using a here doc. Yeah, that's fine then 👍